stl/inc/vector

```diff
@@ -2931,7 +2926,8 @@ public:
         this->_Swap_proxy_and_iterators(_Right);
     }
 
-    _CONSTEXPR20 vector& operator=(vector&& _Right) noexcept(is_nothrow_move_assignable_v<_Mybase>) {
+    _CONSTEXPR20 vector& operator=(vector&& _Right) noexcept(
+        _Choose_pocma_v<_Alvbase> != _Pocma_values::_No_propagate_allocators) {
         if (this == _STD addressof(_Right)) {
             return *this;
         }

```

Comment:
For other reviewers: `vector<bool>::swap` is already unconditionally `noexcept` with a `strengthened` comment in our implementation. We were previously strengthening from nothing to unconditional, after LWG-3778 we're strengthening from conditional to unconditional because we still aren't interested in throwing on precondition violation. (No change requested.)